docs/architectural-decision-records/025-jwt-lambda-authorizer-for-api-access.md

```diff
@@ -0,0 +1,47 @@
+# Use a JWT Lambda Authorizer to enable 3rd party access to our API
```

Comment:
Please write this with the decision record template in the docs folder! Thank you trying to standard things going forward and have dates on things